.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "dev"
+    ]
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! I appreciate what you have done your design however small parts of your design should follow the [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) provided on `requirement`. There your `Log In & Sign Up` page's buttons placed wrongly and implment background color `white` as [Behance](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) 🙏🏼 

![image](https://github.com/AnsarIbrahim/Budget-App/assets/63915024/17b5c527-f2eb-4c83-9e5f-d1da3cfa97f5)